en/config/safety.md

```diff
@@ -0,0 +1,227 @@
+
+The settings and underlying parameters are shown below.
+
+Setting | Parameter | Description
+--- | --- | ---
+RC Loss Timeout | [COM_RC_LOSS_T](../advanced_config/parameter_reference.md#COM_RC_LOSS_T) | Amount of time after losing the RC connection before the failsafe will trigger.
+Failsafe Action | [NAV_RCL_ACT](../advanced_config/parameter_reference.md#NAV_RCL_ACT) | Disabled, Loiter, Return, Land, RC Auto Recovery, Terminate, Lockdown.
+RC Loss Loiter Time | [NAV_RCL_LT](../advanced_config/parameter_reference.md#NAV_RCL_LT) | If the *Failsafe Action* (`NAV_RCL_ACT`) is set to *CASA Outback Challenge rules* this sets the loiter time after RC loss.
+
+<!-- note, does not seem any way to set NAV_RCL_LT in qgc -->
+
+
+### Data Link Loss Failsafe
+
+The Data Link loss failsafe is triggered if a telemetry link is lost.
+
+![Safety - Data Link Loss (QGC)](../../images/qgc/setup/safety_data_link_loss.png)
+
+The settings and underlying parameters are shown below.
+
+Setting | Parameter | Description
+--- | --- | ---
+Data Link Loss Timeout | [COM_DL_LOSS_T](../advanced_config/parameter_reference.md#COM_DL_LOSS_T) | Amount of time after losing the data connection before the failsafe will trigger.
+Failsafe Action | [NAV_DLL_ACT](../advanced_config/parameter_reference.md#NAV_DLL_ACT) | Disabled, Loiter, Return, Land, RC Auto Recovery, Terminate, Lockdown.
+
+
+<!-- ../advanced_config/parameter_reference.md#data-link-loss What are these? 
+Do we explicitly need to recommend not using the CASA rules as the linked parameter does
+-->
+
+
+### Geofence Failsafe
+
+The Geofence is defined as a "virtual" cylinder around the home position.
+If the vehicle moves outside the radius or above the altitude the specified *Failsafe Action* will trigger.
+
+![Safety - Geofence (QGC)](../../images/qgc/setup/safety_geofence.png)
+
+The settings and underlying [geofence parameters](../advanced_config/parameter_reference.md#geofence) are shown below.
+
+Setting | Parameter | Description
+--- | --- | ---
+Action on breach | [GF_ACTION](../advanced_config/parameter_reference.md#GF_ACTION) | None, Warning, Loiter, Return, Flight Terminate.
+Max Radius | [GF_MAX_HOR_DIST](../advanced_config/parameter_reference.md#GF_MAX_HOR_DIST) | Horizontal radius of geofence cylinder. Geofence disabled if 0.
+Max Altitude | [GF_MAX_VER_DIST](../advanced_config/parameter_reference.md#GF_MAX_VER_DIST) | Height of geofence cylinder. Geofence disabled if 0.
+
+
+<!-- flight termination notes: Note: Setting this value to 4 enables flight termination, which will kill the vehicle on violation of the fence. Due to the inherent danger of this, this function is disabled using a software circuit breaker, which needs to be reset to 0 to really shut down the system. WHAT is the breaker, and what happens instead -->
+
+<!--  What are these? Not in QGC
+* GF_ALTMODE (INT32)	- Geofence altitude mode. Comment: Select which altitude reference should be used 0 = WGS84, 1 = AMSL. Values:
```

Comment:
These params are not in QGC:
- Are they used?
- If so, what is GF_SOURCE - ie what does source mean?

Reply:
Yes they are used. With GF_SOURCE you can say whether you want to use the estimated global position or the one directly from the GPS device. Not sure how relevant this is though.